src/backend/marsha/urls.py

```diff
@@ -161,7 +161,7 @@
 
 urlpatterns += [
     re_path(
-        ".*",
+        f"^(?!{settings.STATIC_URL}|{settings.MEDIA_URL}).*",
```

Comment:
I already tried this and this is not enough. If you set `DEBUG=False` in your setting and you fetch a non existing static (`/static/foo.js` for example), you still have a valid response and the route `^(?!/static/|/media/).*` is used.

You can try by adding this 2 lines in the `SiteView` view:
```
logger.info(f"request in catch all: {request.path}")
        logger.info(request.resolver_match.route)
```


```

Reply:
Fact is I forgot to remove the leading slash `/` but now it's fixed, it works as expected, I guess.